pages/ourteam.js

```diff
@@ -0,0 +1,180 @@
+                  </p>
+                </div>
+                <div className="us-two">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/arshi.jpg')}  />
+                    <p className="us-title">Arshi</p>
+                    <p>Software</p>
+                  </p>
+                </div>
+                <div className="us-three">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/shreyas.png')}  />
+                    <p className="us-title">Shreyas</p>
+                    <p>Software</p>
+                  </p>
+                </div>
+              </div>
+              <div className="us">
+                <div className="us-one">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/rickus.jpg')}  />
+                    <p className="us-title">Rickus</p>
+                    <p>Hardware</p>
+                  </p>
+                </div>
+                <div className="us-two">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/sajan.png')}  />
+                    <p className="us-title">Sajan</p>
+                    <p>Hardware</p>
+                  </p>
+                </div>
+                <div className="us-three">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/adam.jpg')}  />
+                    <p className="us-title">Adam</p>
+                    <p>Software/Hardware</p>
+                  </p>
+                </div>
+              </div>
+            </div>
+
+            <div className="ack">
+              <h1>Acknowledgements</h1>
+            </div>
+            <div className="ack">
+              <h1>
+                <Image
+                  className="ack-image"
+                  src={require('../img/sima.jpg')}
+                  width={340}
+                  height={240}
+                />
+              </h1>
+              <Card style={{ width: '40rem' }} bg="#f8f9fa" border="dark">
+                <Card.Body>
+                  <Card.Title>Mihai Sima</Card.Title>
+                  <Card.Subtitle className="mb-2 text-muted">Supervisor</Card.Subtitle>
+                  <Card.Text>
+                  We would like to thank Mihai Sima for
+                   his help with the grading and design.
+                  Professor Mihai Sima specializes in embedded systems and electrical engineering.
```

Comment:
`Dr. Mihai Sima specializes in Embedded Systems and Electrical Engineering, and is a professor at the University of Victoria`